File: Folder tambahan tebak buah/MyApplication/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private ImageView imageView_apel;
    private ImageView imageView_durian;
    private ImageView imageView_ceri;
    private ImageView imageView_alpukat;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        setInisialisasi();
        onClickMantab();
    }

    private void onClickMantab() {
        imageView_durian.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent d = new Intent(MainActivity.this, TebakActivity.class);
                d.putExtra("nama_buah", "durian");
                startActivity(d);
            }
        });
        imageView_ceri.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent c = new Intent(MainActivity.this, TebakActivity.class);
                c.putExtra("nama_buah", "ceri");
                startActivity(c);
            }
        });
        imageView_apel.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent a = new Intent(MainActivity.this, TebakActivity.class);
                a.putExtra("nama_buah", "apel");
                startActivity(a);
            }
        });
        imageView_alpukat.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent al = new Intent(MainActivity.this, TebakActivity.class);
                al.putExtra("nama_buah", "alpukat");
                startActivity(al);
            }
        });
    }

    private void setInisialisasi() {
        imageView_alpukat = findViewById(R.id.imageView_alpukat);
        imageView_apel = findViewById(R.id.imageView_apel);
        imageView_ceri = findViewById(R.id.imageView_ceri);
        imageView_durian = findViewById(R.id.imageView_durian);
    }
}

File: AplikasiTebakBuah/app/src/main/java/com/example/aplikasitebakbuah/MainActivity.kt
package com.example.aplikasitebakbuah

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var imageView_apel: ImageView
    private lateinit var imageView_durian: ImageView
    private lateinit var imageView_ceri: ImageView
    private lateinit var imageView_alpukat: ImageView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setInisialisasi()
        onClickMantab()
    }

    private fun onClickMantab() {
        imageView_durian.setOnClickListener {
            val d = Intent(this@MainActivity, TebakActivity::class.java)
            d.putExtra("nama_buah", "durian")
            startActivity(d)
        }
        imageView_ceri.setOnClickListener {
            val c = Intent(this@MainActivity, TebakActivity::class.java)
            c.putExtra("nama_buah", "ceri")
            startActivity(c)
        }
        imageView_apel.setOnClickListener {
            val a = Intent(this@MainActivity, TebakActivity::class.java)
            a.putExtra("nama_buah", "apel")
            startActivity(a)
        }
        imageView_alpukat.setOnClickListener {
            val al = Intent(this@MainActivity, TebakActivity::class.java)
            al.putExtra("nama_buah", "alpukat")
            startActivity(al)
        }
    }

    private fun setInisialisasi() {
        imageView_alpukat = findViewById(R.id.imageView_alpukat)
        imageView_apel = findViewById(R.id.imageView_apel)
        imageView_ceri = findViewById(R.id.imageView_ceri)
        imageView_durian = findViewById(R.id.imageView_durian)
    }
}
